core/src/main/resources/lib/hudson/propertyTable.jelly

```diff
@@ -30,8 +30,24 @@ THE SOFTWARE.
     <st:attribute name="items" use="required">
       A Map object that gets rendered as a table.
     </st:attribute>
+    <st:attribute name="sensitive" use="optional">
+      Set to true if the information shown in the table is sensitive and should be hidden by default. Since TODO.
+    </st:attribute>
   </st:documentation>
-  <table class="jenkins-table sortable">
+  <j:if test="${attrs.sensitive}">
+    <st:adjunct includes="lib.hudson.property-table"/>
+  </j:if>
+  <j:set var="tableId" value="${h.generateId()}"/>
+  <j:if test="${attrs.sensitive}">
+    <div class="jenkins-!-margin-bottom-2">
+      ${%redactedBlurb}
+      <button class="all-hidden-info jenkins-button" data-table-id="${tableId}">
+        <l:icon src="symbol-search"/>
```

Comment:
```suggestion
        <l:icon src="symbol-view"/>
```
Any objections about using the eye symbol? It's used in different locations to highlight something can be looked at.

Reply:
Great suggestion, thanks!